tests/test_batch_correction.py

```
@@ -0,0 +1,42 @@
+        "comment": "#",
+        "sep": "\t",
+        "output": TESTS_DIR / "example/ibaq_corrected_combined.tsv",
+        "sample_batch_map": None,
+        "sample_id_column": "SampleID",
+        "protein_id_column": "ProteinName",
+        "ibaq_column": "Ibaq"
+    }
+    run_batch_correction(**args)
+    
+    # Test invalid sample IDs
+    with pytest.raises(ValueError):
```

Ruff (0.8.2)

Undefined name `pytest`

(F821)